### ISSUE TITLE
ERROR: Check failed: img_height >= crop_size (0 vs. 227)

### ISSUE THREAD

**Issue by mrgloom** [NVIDIA/DIGITS#1079]
I get some strange error `ERROR: Check failed: img_height >= crop_size (0 vs. 227)` after successfully training net for ~1.5h

~~~
Ignoring source layer train-data
Test net output #0: accuracy = 0.934929
Test net output #1: loss = 0.160105 (* 1 = 0.160105 loss)
Iteration 12870 (1.54456 iter/s, 25.2499s/39 iter), loss = 0.103487
Train net output #0: loss = 0.0928845 (* 1 = 0.0928845 loss)
Iteration 12870, lr = 0.00314696
Iteration 12909 (2.87442 iter/s, 13.5679s/39 iter), loss = 0.128901
Train net output #0: loss = 0.200622 (* 1 = 0.200622 loss)
Iteration 12909, lr = 0.0031262
Iteration 12948 (2.8783 iter/s, 13.5497s/39 iter), loss = 0.0660827
Train net output #0: loss = 0.0832538 (* 1 = 0.0832538 loss)
Iteration 12948, lr = 0.00310543
Iteration 12987 (2.90467 iter/s, 13.4267s/39 iter), loss = 0.0672469
Train net output #0: loss = 0.0573448 (* 1 = 0.0573448 loss)
Iteration 12987, lr = 0.00308466
Iteration 13026 (2.80167 iter/s, 13.9203s/39 iter), loss = 0.097256
Train net output #0: loss = 0.134702 (* 1 = 0.134702 loss)
Iteration 13026, lr = 0.0030639
Could not decode datum
Check failed: img_height >= crop_size (0 vs. 227)
~~~

**Comment by mrgloom** [comment]
Seems some of images in lmdb broken?

~~~
I0917 12:42:49.286123  3359 net.cpp:723] Ignoring source layer train-data
I0917 12:43:00.713511  3359 solver.cpp:429]     Test net output #0: accuracy = 0.934929
I0917 12:43:00.713564  3359 solver.cpp:429]     Test net output #1: loss = 0.160105 (* 1 = 0.160105 loss)
I0917 12:43:14.140091  3359 solver.cpp:242] Iteration 12870 (1.54456 iter/s, 25.2499s/39 iter), loss = 0.103487
I0917 12:43:14.140151  3359 solver.cpp:261]     Train net output #0: loss = 0.0928845 (* 1 = 0.0928845 loss)
I0917 12:43:14.140169  3359 sgd_solver.cpp:106] Iteration 12870, lr = 0.00314696
I0917 12:43:27.708305  3359 solver.cpp:242] Iteration 12909 (2.87442 iter/s, 13.5679s/39 iter), loss = 0.128901
I0917 12:43:27.717725  3359 solver.cpp:261]     Train net output #0: loss = 0.200622 (* 1 = 0.200622 loss)
I0917 12:43:27.717767  3359 sgd_solver.cpp:106] Iteration 12909, lr = 0.0031262
I0917 12:43:41.267568  3359 solver.cpp:242] Iteration 12948 (2.8783 iter/s, 13.5497s/39 iter), loss = 0.0660827
I0917 12:43:41.267633  3359 solver.cpp:261]     Train net output #0: loss = 0.0832538 (* 1 = 0.0832538 loss)
I0917 12:43:41.267652  3359 sgd_solver.cpp:106] Iteration 12948, lr = 0.00310543
I0917 12:43:54.694509  3359 solver.cpp:242] Iteration 12987 (2.90467 iter/s, 13.4267s/39 iter), loss = 0.0672469
I0917 12:43:54.694571  3359 solver.cpp:261]     Train net output #0: loss = 0.0573448 (* 1 = 0.0573448 loss)
I0917 12:43:54.694588  3359 sgd_solver.cpp:106] Iteration 12987, lr = 0.00308466
I0917 12:44:08.615037  3359 solver.cpp:242] Iteration 13026 (2.80167 iter/s, 13.9203s/39 iter), loss = 0.097256
I0917 12:44:08.620816  3359 solver.cpp:261]     Train net output #0: loss = 0.134702 (* 1 = 0.134702 loss)
I0917 12:44:08.620857  3359 sgd_solver.cpp:106] Iteration 13026, lr = 0.0030639
libpng error: IDAT: CRC error
E0917 12:44:11.229673  3367 io.cpp:173] Could not decode datum
F0917 12:44:11.229857  3367 data_transformer.cpp:479] Check failed: img_height >= crop_size (0 vs. 227)
*** Check failure stack trace: ***
@     0x7f8ddd9c0daa  (unknown)
@     0x7f8ddd9c0ce4  (unknown)
@     0x7f8ddd9c06e6  (unknown)
@     0x7f8ddd9c3687  (unknown)
@     0x7f8dddfb27e0  caffe::DataTransformer<>::TransformPtr()
@     0x7f8dddfbc460  caffe::DataTransformer<>::TransformPtrEntry()
@     0x7f8dddfec126  boost::detail::function::void_function_obj_invoker0<>::invoke()
@     0x7f8dddfec5b9  ThreadPool::main_loop()
@     0x7f8ddc294a4a  (unknown)
@     0x7f8ddc073184  start_thread
@     0x7f8ddc59937d  (unknown)
@              (nil)  (unknown)
~~~

**Comment by lukeyeager** [comment]
Yes, that looks like one of the images in your LMDB is corrupted. Did  you create the LMDB with DIGITS? Are there any warnings or errors in your create_db log?

**Comment by mrgloom** [comment]
Yes I have created db with DIGITS and successfully trained about 5 networks, after that db corrupted. Create logs have no warning or errors.

I close issue, because this seems not DIGITS issue.
